src/util.h

```diff
@@ -178,77 +178,75 @@ inline TypeName* Unwrap(v8::Local<v8::Object> object);
 
 inline void SwapBytes(uint16_t* dst, const uint16_t* src, size_t buflen);
 
-class Utf8Value {
+// Allocates an array of member type T. For up to StackStorageSize items,
+// the stack is used, otherwise malloc().
+template<typename T, size_t StackStorageSize>
```

Comment:
Style: space after `template` and can you call the second parameter `kStackStorageSize`?